src/state/queries/suggested-follows.ts

```diff
@@ -31,7 +31,7 @@ const suggestedFollowsByActorQueryKey = (did: string) => [
   did,
 ]
 
-export function useSuggestedFollowsQuery() {
+export function useSuggestedFollowsQuery(options?: {limit?: number}) {
```

Comment:
Probably gotta update the query key to include the param

Reply:
Good call